src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/wwwroot/css/site.css

```diff
@@ -165,6 +165,7 @@ app {
     .main .top-row {
         position: sticky;
         top: 0;
+        z-index: 9999;
```

Comment:
Please choose a minimal z-index, which will serve the need. We want to avoid assigning random big numbers.
Also, consider applying the same change to the other (web-assembly) project template.

Reply:
I've update it to `z-index: 1` and made the same change [here](https://github.com/deisterhold/aspnetcore/blob/patch-1/src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/Client/wwwroot/css/site.css#L168).

![Example](https://user-images.githubusercontent.com/2267691/78825248-40167600-79a5-11ea-824e-6193bf2a3d6b.png)